### ISSUE TITLE
What doesthe accuracy refer to ? 

### ISSUE THREAD

**Issue by farshidrayhan** [kuangliu/pytorch-cifar#54]
What doesthe accuracy refer to ?
Train score or test score ?

I have tested with Resnet34. The test accuracy score isnt nearly close enough to the reported one.

Can someone please clarify ?

**Comment by sh0416** [comment]
In my case, the accuracy in README.md is test accuracy.

You need to adjust learning rate (0.1, 0.01, 0.001), which is critical to improve the accuracy.

I tested resnet101 and the final test accuracy is 93.800%, train accuracy is almost 99%.

**Comment by farshidrayhan** [comment]
Thank you for clearing that up.

I just needed to clear that up. I just needed to train a bit more.
